packages/utxo-lib/src/compose/index.ts

```diff
@@ -4,6 +4,7 @@ import * as transaction from './transaction';
 import { coinselect } from './coinselect';
 
 import type { Result } from './coinselect';
+import { formatStringFeeRate } from '../coinselect/utils';
```

Comment:
pls, move this import to line 5 (before import type) or even better, join imports from `./coinselect` (line 4 + line 6) - looks ilke me and the reviewer didn't noticed that while moving this lib from the origin :)

Reply:
done 463ab037d9c46a43792a5ad5b0e6f993c4cf7d1d

Reply:
almost :)
should be:
```
import { coinselect, Result } from './coinselect';
```

Reply:
Or do you want me to just get rid of the `type`?

Reply:
i rather do, i've looked at code and we are not using it anywhere, lets be consistent

Reply:
226eda709e01d863dee3d9d87b8a9fd608a0e01e